frontend/src/lib/context/outliner.tsx

```diff
@@ -183,6 +184,19 @@ export const OutlinerProvider = ({
         }
     }, [query.data, setViewState]);
 
+    useEffect(() => {
```

Comment:
Since the goal is to do it at startup, I believe it's best to do it in a `useEffect` in the `App` context and make it dependent on the startup query - that way it only runs once after getting the startup response. Here, in the outliner context, it would run per each outliner.